README.md

```diff
@@ -191,6 +191,12 @@ functional tests using the [pulp_smash](https://pulp-smash.readthedocs.io/en/lat
 In order to use these tests, you will need to address the "FIXME" messages left in places where
 plugin-writer intervention is required.
 
+At this point, you have a one off chance to use `--all` instead of `--bootstrap` to include all the
+other sections described below.
+
+  **Note** : Regenerating the *bootstrap* section at a later time will reset all files to their
+  original state, which is almost always not intended.
```

Comment:
Nice!

Reply:
I think we need to expand upon this a little bit. Can we try and elaborate on what `--all` means here, rather than saying it includes other sections described below? If I use --bootstrap, I get a minimum viable plugin that I can then customize(I'm guessing), if I run `--all`, what specifically do I get? 
The fact that this is an option means that someone might want to use it. Who is this person and what would be their aim? Could we add an example of when it would be an advantage to use `--all` rather than `--bootstrap` at this point, since it is their one chance to do this?

Reply:
```
  --generate-config     Create or update a plugin template config for a plugin and exit.
                        
  --bootstrap           Create a new plugin and template boilerplate code.
                        
  --test                Generate or update functional and unit tests.
                        
  --travis              Generate or update travis configuration files.
                        
  --docs                Generate or update plugin documentation.
                        
```
you can start a plugin interactively:
0 - `./plugin-template --generate-config --plugin-app-label catdog pulp_catdog`
1 - `./plugin-template --generate-config --plugin-app-label catdog pulp_catdog`
with the previous steps, you can have a minimum viable plugin, and choose whether you add tests, CI or docs.
**Note:** step 0 is mandatory, as plugin_template depends on the config
So, if a week later I want to add tests I can:
2 - `./plugin-template --test pulp_catdog`

If the plugin writer wants a full plugin since the beginning, `--all` would be useful as it executes (bootstrap, test, travis, docs). In our CI we want to build a full plugin for testing, so we use `--all`:
https://github.com/pulp/plugin_template/blob/master/.travis/bootstrap_catdog.sh

Reply:
How about something like: 

> At this point, you have a one-off opportunity to use the --all option, which generates everything included in the --bootstrap option, as well as documentation, functional and unit test, and travis configuration file templates that you require to support a plugin.